sdk/storage/src/datalake/path_client.cpp

```diff
@@ -12,8 +12,89 @@
 #include "datalake/datalake_utilities.hpp"
 #include "http/curl/curl.hpp"
 
+#include <limits>
+#include <utility> //std::pair
+
 namespace Azure { namespace Storage { namespace DataLake {
 
+  std::string GetSubstringTillDelimiter(
```

Comment:
put it into anonymous namespace if it's only used in this file?

Reply:
I'll change this. But Jason and I used to fight over this when doing other work. Let's sync offline.